### ISSUE TITLE
questions regarding code details

### ISSUE THREAD

**Issue by NIRVANALAN** [sunfanyunn/vGraph#8]
Hi fanyun,
Thanks for sharing the code. I have several question here
1. after acquiring the estimated community membership embedding ```new_z```, according to the paper, shouldn't we calculate the softmax p(c|z=j), what's the meaning of ```decoder```? To approximate the calculation?
2. no lr decay found in nonoverlap.py; did you use it when training like ```CORA``` for nonoverlap community detection?

**Comment by lif323** [comment]
Hi, for the first question, I think new_z comes from the sampling of distribution q(z|w, c). But it is expressed in the form of embeddding. The function of decoder is to obtain p(c|z), where z~q(z|w, c), but p(c|z) is not normailzed.
I have the same doubts about the second question.
